src/scrapers/mizrahi.ts

```diff
@@ -211,7 +211,7 @@ class MizrahiScraper extends BaseScraperWithBrowser<ScraperSpecificCredentials>
     await waitUntilElementFound(this.page, `a[href*="${TRANSACTIONS_PAGE}"]`);
     await this.page.$eval(`a[href*="${TRANSACTIONS_PAGE}"]`, (el) => (el as HTMLElement).click());
 
-    const accountNumberElement = (await this.page.$$('#AccountPicker b'))[0];
+    const accountNumberElement = (await this.page.$$('#AccountPicker'))[0];
```

Comment:
It indeed solved the problem, but the `accountNumber` turned out to be empty for me.

Instead, I tested this and it worked:

```suggestion
    const accountNumberElement = (await this.page.$$('#dropdownBasic b'))[0];
```